core/chainlink.devspace.Dockerfile

```diff
@@ -38,7 +38,7 @@ FROM golang:1.21-bullseye
 
 ARG CHAINLINK_USER=chainlink
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y ca-certificates gnupg lsb-release curl
+RUN apt-get update && apt-get install -y ca-certificates gnupg lsb-release curl patchelf
```

Comment:
Should patchelf only be installed on the goreleaser built docker images?

Reply:
Sergey mentioned his the slack thread that he'd like this - will double-check with him.